tests/eval_files/async101.py

```diff
@@ -1,4 +1,4 @@
-# NOASYNCIO
+# ASYNCIO_NO_ERROR - no nursery/cancelscope in asyncio
```

Comment:
Oh right right, `trio.[move_on/fail]_[after/at]` are just wrapper functions for cancelscopes.
It might still be good (and fairly easy) to raise errors for an asyncio user doing it, even if our real suggestion is for them to switch to anyio.

Secret plan is for asyncio users to start running our checks, then when getting tons of useful errors realize they should switch to anyio :smiling_imp: